src/module_2/explicacion_data_analysis.md

```diff
@@ -0,0 +1,425 @@
+
+ #make_corr_matrix_and_statictics(df_orders_combined_inventory)
+ print(df_orders_combined_inventory.head()) 
+```
+
+           variant_id  number_of_orders  purchase_probability  price  \
+    0  34081589887108              4487              0.511456  10.79   
+    1  34284951863428              1133              0.129146   3.69   
+    2  34284950356100               954              0.108743   1.99   
+    3  34370361229444               939              0.107033   4.99   
+    4  33826465153156               884              0.100764   1.89   
+    
+       compare_at_price         vendor                     product_type  \
+    0             11.94          oatly       long-life-milk-substitutes   
+    1              3.99         plenty  toilet-roll-kitchen-roll-tissue   
+    2              3.00          fairy                      dishwashing   
+    3              5.50  whogivesacrap  toilet-roll-kitchen-roll-tissue   
+    4              1.99    clearspring              tins-packaged-foods   
+    
+                                            tags  
+    0                          [oat-milk, vegan]  
+    1                             [kitchen-roll]  
+    2  [discontinue, swapped, washing-up-liquid]  
+    3                [b-corp, eco, toilet-rolls]  
+    4    [gluten-free, meat-alternatives, vegan]  
+
+
+Aqui vemos como he calculado una purchase probability contando en numero de orders y dividiendo el numero de orders en que aparece cada articulo por este total
+
+
+```python
+make_corr_matrix_and_statictics(df_orders_combined_inventory)
+```
+
+             variant_id  number_of_orders  purchase_probability        price  \
+    count  1.477000e+03       1477.000000           1477.000000  1477.000000   
+    mean   3.690916e+13         62.532837              0.007128     6.553785   
+    std    2.695602e+12        151.179244              0.017232     6.490753   
+    min    3.361529e+13          1.000000              0.000114     0.570000   
+    25%    3.427657e+13          9.000000              0.001026     2.890000   
+    50%    3.927260e+13         27.000000              0.003078     4.490000   
+    75%    3.946662e+13         71.000000              0.008093     7.990000   
+    max    4.015515e+13       4487.000000              0.511456    59.990000   
+    
+           compare_at_price  
+    count       1477.000000  
+    mean           7.316168  
+    std            7.071117  
+    min            0.750000  
+    25%            3.000000  
+    50%            4.990000  
+    75%            8.970000  
+    max           60.000000  
+
+
+
+    
+![png](explicacion_data_analysis_files/explicacion_data_analysis_13_1.png)
+    
+
+
+En esta corr matrix yo no veo mucho y seguramente sea debido a que habia muchos products ids que aparecian en orders pero no en el inventario por lo que ahi perdiamos mucha info a la hora de mergear ambis( opté por no usarlos)
```

Comment:
I would not be expecting to gain much info from the corr of these variables.

Something that immediately should draw your attention are the correlation os value 1. What does it mean that number_of_ordres correaltes 1 with purchase probability?